eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -93,11 +133,18 @@ catch {
 $resp | Write-Verbose
 
 if ($resp.Count -gt 0) {
-    Write-Host -f green "Pull request already exists $($resp[0].html_url)"
+  try {
+    LogDebug "Pull request already exists $($resp[0].html_url)"
```

Comment:
We should keep this a normal log message otherwise it will not show up in Devops logging unless you set System.Debug = true

Reply:
I tested it here and it does show up https://dev.azure.com/azure-sdk/playground/_build/results?buildId=551403&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=24311d03-63ec-5098-c8e5-7df09da2b6d4